app/models/form/lettings/pages/rent_weekly.rb

```diff
@@ -4,30 +4,8 @@ def initialize(id, hsh, subsection)
     @id = "rent_weekly"
     @header = "Household rent and charges"
     @depends_on = [
-      { "period" => 1, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 1, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 5, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 5, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 6, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 6, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 7, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 7, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 8, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 8, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 9, "household_charge" => 0, "is_carehome" => 0 },
-      { "period" => 9, "household_charge" => nil, "is_carehome" => 0 },
-      { "period" => 1, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 1, "household_charge" => nil, "is_carehome" => nil },
-      { "period" => 5, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 5, "household_charge" => nil, "is_carehome" => nil },
-      { "period" => 6, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 6, "household_charge" => nil, "is_carehome" => nil },
-      { "period" => 7, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 7, "household_charge" => nil, "is_carehome" => nil },
-      { "period" => 8, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 8, "household_charge" => nil, "is_carehome" => nil },
-      { "period" => 9, "household_charge" => 0, "is_carehome" => nil },
-      { "period" => 9, "household_charge" => nil, "is_carehome" => nil },
+      { "rent_and_charges_paid_weekly?" => true, "household_charge" => 0, "is_carehome?" => false },
```

Comment:
There are various other places that `"is_carehome" => nil or 0` could be replaced with `"is_carehome?" => false`, for example in `app/models/form/lettings/pages/rent_4_weekly.rb`. I'd recommend either doing that as part of this ticket, or raising a new tech debt ticket.

Reply:
> ok I relented on this in a big way

What do you mean?